examples/webgpu_postprocessing_pixel.html

```diff
@@ -0,0 +1,290 @@
+			const texChecker2 = pixelTexture( loader.load( 'textures/checker.png' ) );
+			texChecker.repeat.set( 3, 3 );
+			texChecker2.repeat.set( 1.5, 1.5 );
+
+			// meshes
+
+			const boxMaterial = new THREE.MeshPhongMaterial( { map: texChecker2 } );
+
+			function addBox( boxSideLength, x, z, rotation ) {
+
+				const mesh = new THREE.Mesh( new THREE.BoxGeometry( boxSideLength, boxSideLength, boxSideLength ), boxMaterial );
+				mesh.castShadow = false;
+				mesh.receiveShadow = true;
+				mesh.rotation.y = rotation;
+				mesh.position.y = boxSideLength / 2;
+				mesh.position.set( x, boxSideLength / 2 + .0001, z );
+				scene.add( mesh );
+				return mesh;
+
+			}
+
+			addBox( .4, 0, 0, Math.PI / 4 );
+			addBox( .5, - .5, - .5, Math.PI / 4 );
+
+			const planeSideLength = 2;
+			const planeMesh = new THREE.Mesh(
+				new THREE.PlaneGeometry( planeSideLength, planeSideLength ),
+				new THREE.MeshPhongMaterial( { map: texChecker } )
+			);
+			planeMesh.receiveShadow = true;
+			planeMesh.rotation.x = - Math.PI / 2;
+			scene.add( planeMesh );
+
+			const radius = .2;
+			const geometry = new THREE.IcosahedronGeometry( radius );
+			crystalMesh = new THREE.Mesh(
+				geometry,
+				new THREE.MeshPhongMaterial( {
+					color: 0x68b7e9,
+					emissive: 0x4f7e8b,
+					shininess: 10,
+					specular: 0xffffff
+				} )
+			);
+			crystalMesh.receiveShadow = false;
+			crystalMesh.castShadow = false;
+			scene.add( crystalMesh );
+
+			// lights
+
+			const directionalLight = new THREE.DirectionalLight( 0xfffecd, 1.5 );
+			directionalLight.position.set( 100, 100, 100 );
+			directionalLight.castShadow = true;
+			directionalLight.shadow.mapSize.set( 2048, 2048 );
+			scene.add( directionalLight );
+
+			const fillLight = new THREE.DirectionalLight( 0x2a3c6b, 1.2 );
+			fillLight.position.set(-100, 100, 100);
+			fillLight.castShadow = true;
+			fillLight.shadow.mapSize.set( 2048, 2048 );
+			scene.add( fillLight );
+
+			const spotLight = new THREE.SpotLight( 0xffc100, 10, 10, Math.PI / 16, .02, 2 );
+			spotLight.position.set( 2, 2, 0 );
+			const target = spotLight.target;
+			scene.add( target );
+			target.position.set( 0, 0, 0 );
+			spotLight.castShadow = true;
+			scene.add( spotLight );
+
+			renderer = new THREE.WebGPURenderer({ antialias: false });
+			renderer.shadowMap.enabled = true;
+			//renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( window.innerWidth, window.innerHeight );
+			renderer.setAnimationLoop( animate );
+			document.body.appendChild( renderer.domElement );
+
+			effectController = {
+				pixelSize: uniform( 14 ),
+				normalEdgeStrength: uniform( 0.3 ),
+				depthEdgeStrength: uniform( 0.4 ),
+				pixelAlignedPanning: true
+			};
+
+			postProcessing = new THREE.PostProcessing( renderer );
+
+			const scenePass = pass( scene, camera );
+			scenePass.setMRT( mrt( {
+				output: output,
+				normal: directionToColor(normalView),
```

Comment:
This should be just `normal: normalView`.

Reply:
That certainly makes sense to me, although for some reason when testing, the output of directionToColor(normalView) matched the normal output of RenderPixelatedPass while normalView did not. That never seemed right to me though, so I'll go back and check.

Reply:
I've reverted back to normalView, but the effect of normalEdgeStrength is noticeably different/weaker compared to using directionToColor(normalView). I'll add some images to show what I mean, but I believe in this instance, to replicate the effect properly, directionToColor( normalView ) is the correct way our normal pass needs to be configured.

WebGL Pixelation tNormal texel output:
<img width="767" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/311106bc-69b7-4276-ba68-47baab33351d">

WebGL Pixelation Output ( Max NormalEdgeStrength)
<img width="783" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/d2f31cc0-05f9-4b8e-a935-b4bf9a3f25fc">

WebGPU Pixelation normalView texel output ( i.e the raw output of the scenePass's normal texture node when the normal render target is set to normalView ):
<img width="787" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/74492d4e-5bf0-4bdc-bf5b-3b74225c8458">

WebGPU Pixelation Output ( Max NormalEdgeStrength with normalView as scenePass normal output )
// Note the complete lack of any edge detection on the Icosahedron
<img width="787" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/4f7b90cb-0fc6-437e-9c9b-fe67af7977b7">

WebGPU Pixelation directionToColor( normalView ) texel output
// Image is not pixelized since the resolution is adjusted as a post-step in this implementation
<img width="785" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/b667942f-9bb3-45bb-8d5e-3c6fcc593e1a">

WebGPU Pixelation Output ( Max NormalEdgeStrength with directionToColor( normalView ) as scenePass normal output )
<img width="772" alt="image" src="https://github.com/mrdoob/three.js/assets/62450112/4a057dbf-e95c-4b16-972b-e25a3c47564a">

Reply:
These are roughly analagous to the same effects I noticed before, which is why I ultimately chose directionToColor( normalView ) over normalView, even if intuition would lead us to use normalView.